assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,171 @@
-console.log('***** Music Collection *****')
+console.log("***** Music Collection *****");
+// ---------------
+// Data
+// ---------------
+import { tracksData, testTracks, testTracks7 } from "../utils/tracksData.js";
+import { formatDuration } from "../utils/formatDuration.js";
+
+// collection array
+let collection = [];
+
+// ---------------
+// Functions
+// ---------------
+
+// add to collection function
+const addToCollection = (title, artist, yearPublished, tracks) => {
+  const addition = { title, artist, yearPublished, tracks };
+  collection.push(addition);
+  return addition;
+};
+
+// show collection function
+const showCollection = (array) => {
```

Comment:
Great variable names throughout this project, and then here is this "array" just sitting here. Perhaps.... "albums"?